packages/common/src/intl/locales/pt.json

```diff
@@ -15,5 +15,6 @@
   "app.transactions": "Transactions",
   "app.welcome": "Hello, {name}!",
   "breadcrumb.item": "#{id}",
-  "button.ok": "ok"
-}
+  "button.ok": "ok",
+  "button.open-the-menu": "Abra o menu"

```

Comment:
I also can see having to put the key in all translation files to get tedious quickly?

Reply:
I'm already over it.
Started with the approach similar to mobile, where `en` is the base and you just add in parts that you have of other languages. Missing translations fall back to the base language. had trouble getting that to fly with lazy loading and decided that maybe adding the key to all translation files was ok.